client/controllers/bookHandler/bookHandler.go

```diff
@@ -0,0 +1,25 @@
+package controllers
```

Comment:
nit: rename `bookHandler` directory to `book`

Reply:
and rename `bookHandler.go` to `book_handler.go` or `interface.go`

Reply:
I made mistake, It was supposed to be named book_history. Fixed